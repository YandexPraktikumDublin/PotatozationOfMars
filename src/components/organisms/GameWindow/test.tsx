import React from 'react'
import { shallow } from 'enzyme'
import toJSON from 'enzyme-to-json'

import { GameWindow } from '.'

describe('<GameWindow />', () => {
  it('should renders correct <GameWindow />', () => {
    const wrapper = shallow(<GameWindow />)

    expect(toJSON(wrapper)).toMatchSnapshot()
  })
})
