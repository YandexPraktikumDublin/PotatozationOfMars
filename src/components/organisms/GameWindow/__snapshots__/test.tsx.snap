// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<GameWindow /> should renders correct <GameWindow /> 1`] = `
<div
  className="relative flex justify-center items-center"
>
  <GameCanvas />
  <NavigationButton
    className="absolute top-3 right-3"
    imageSrc={
      <svg
        data-file-name="SvgFullscreenOn"
      />
    }
    onClick={[Function]}
    title="Full screen"
  />
</div>
`;
